policyengine_canada/parameters/gov/provinces/nl/tax/income/supplement/income_supplement/additional_amount/max_amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Newfoundland and Labrador provides the following income supplement maximum additional amount.
+values:
+  2022-12-31: 266
+metadata:
+  unit: currency-CAD
+  label: Newfoundland and Labrador Income Supplement max addition amount
+  reference:
+    - title: Government of Canada - Newfoundland and Labrador Income Supplement
+      href: https://www.gov.nl.ca/fin/tax-programs-incentives/personal/income-supplement/
+    - title: Government of Canada - Newfoundland and Labrador Income Tax Act
```

Comment:
could you add the paragraph where this program can be found

Reply:
in all yaml files

Reply:
If you click on the link, it should direct you to the income supplement part.